recipes/turtle/all/conanfile.py

```diff
@@ -16,25 +15,16 @@ class TurtleConan(ConanFile):
     def _source_subfolder(self):
         return "source_subfolder"
 
-    _cmake = None
-
-
     def requirements(self):
         self.requires("boost/1.73.0")
 
+    def package_id(self):
+        self.info.header_only()
+
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version], destination = self._source_subfolder)
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename(self.name + "-" + self.version, self._source_subfolder)
 
     def package(self):
-        self.copy("**/*.hpp", dst=".", src=self._source_subfolder, keep_path=True)
-        tools.download(
```

Comment:
It would be nice if you could describe such changes shortly in the PR so that reviews don't have to find out everything be themselves.

You (probably) changed the download of the official manual release downloads to the GitHub automatic source download archives, because it contains the LICENSE file and we can remove those workarounds for download manually the LICENSE file.

Reply:
Correct ;)